### PR TITLE
Week2

### PR DESCRIPTION
**Pull request by ofeterovskaya**

### CODE REVIEW

File: 01-node-tutorial/answers/16-streams.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  encoding: 'utf8',
+  highWaterMark: 200
+});
+
+stream.on('data', (msg) => {
+  counter++;
+  console.log(msg);
+  if (counter >= 10) { // stop the stream after 10 chunks
+    stream.destroy();
+  }
```
Comment (mattcarlotta) [inline 01-node-tutorial/answers/16-streams.js:14]: Nitpick (doesn't require changes):
I'm not sure why this code is here as you'll typically want to read the entire stream and not destroy it after 10 chunks!

File: 01-node-tutorial/answers/package.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    "picomatch": "^2.3.1",
+    "pstree.remy": "^1.1.8",
+    "readdirp": "^3.6.0",
+    "semver": "^7.6.0",
+    "simple-update-notifier": "^2.0.0",
+    "supports-color": "^5.5.0",
+    "to-regex-range": "^5.0.1",
+    "touch": "^3.1.0",
+    "undefsafe": "^2.0.5",
+    "yallist": "^4.0.0"
```
Comment (mattcarlotta) [inline 01-node-tutorial/answers/package.json:38]: Nitpick (doesn't require changes):
I'm not sure why these other dependencies are here, but you should only need to have the one dev dependency (nodemon)!

File: 01-node-tutorial/answers/writeWithPromisesAwait.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+const { writeFile, readFile } = require("fs").promises;
+
+const writer = async () => {
+  try {
+    // Write 'Hello  to temp.txt, overwriting any existing content
+    await writeFile('./temporary/temp.txt', 'Hello \n', { flag: 'a' });
+    // Append 'Nice to see you  to temp.txt
+    await writeFile('./temporary/temp.txt', 'Nice to see you \n', { flag: 'a' });
+    // Append 'Bye, see you again' to temp.txt
+    await writeFile('./temporary/temp.txt', 'Bye, see you again', { flag: 'a' });
```
Comment (mattcarlotta) [inline 01-node-tutorial/answers/writeWithPromisesAwait.js:11]: Great work, you nailed this code! This is exactly what the assignment was looking for! 🥳

**Review by mattcarlotta — APPROVED** [review]
I have a few nitpicks below, but nothing that prevents this assignment from being approved. You absolutely knocked this assignment out of the park! Great work Oksana! 🥳 🚀 🎉

File: 01-node-tutorial/answers/16-streams.js (hunk 1 above)
Comment (ofeterovskaya) [reply 01-node-tutorial/answers/16-streams.js:14]: oh. I did it for myself when played and forgot to remove it

File: 01-node-tutorial/answers/package.json (hunk 2 above)
Comment (ofeterovskaya) [reply 01-node-tutorial/answers/package.json:38]: Only nodemon dependency was there the only one, but during one of mentor session someone's idea was to run another code and I happened to get all of these :)

**Pull request merged**